### ISSUE TITLE
Padding with zeroes 

### ISSUE THREAD

**Issue by fedek6** [HubSpot/odometer#67]
Hi,
Maybe you should add padding with zeroes option? It would be more like a real world odometer. 
![r295663_1271558](https://cloud.githubusercontent.com/assets/6105281/3834925/f8ffede6-1db6-11e4-8696-2152a8301682.jpg)

**Comment by d4nyll** [comment]
Yes, even putting a string like '0033' doesn't work. This is definitely an important feature!

**Comment by jshamley** [comment]
agreed, I'd be interested in this feature too.

**Comment by theother1** [comment]
yes please! leading zeros are a MUST

:)

**Comment by spiceygas** [comment]
Inside the odometer javascript file, look for the render() function.  At the end of the function you'll see:

      for (_j = 0, _len1 = _ref.length; _j < _len1; _j++) {
        digit = _ref[_j];
        if (digit === '.') {
          wholePart = true;
        }
        this.addDigit(digit, wholePart);
      }
	  
      return this.startWatchingMutations();

Between the end of the for loop and the return statement, add the following code.  Replace **MIN_INTEGER_LEN** with whatever you want as the minimum length.

	  if (MIN_INTEGER_LEN > this.digits.length)
	  {
		var initialLength;
		initialLength = this.digits.length;
		for (_j = 0; _j < MIN_INTEGER_LEN - initialLength; _j++)
		{
			this.addDigit(0,true);
		}
	  }


For example, if you use MIN_INTEGER_LENGTH = 8 and your odometer is 12,345 then you'll display 00,012,345.  The stylesheets still work (the project has a couple to choose between).

![zero_padded](https://cloud.githubusercontent.com/assets/11875239/7073250/c349112e-deb7-11e4-963b-65c660842f04.png)

**Comment by KoltonG** [comment]
@spiceygas this only works if the numbers are not changing, when update is called and the numbers begins to transition the added 0's go away. Any idea how to fix this?

**Comment by adamschwartz** [comment]
See https://github.com/HubSpot/odometer/issues/106 for a workaround.

**Comment by spiceygas** [comment]
@KoltonG I'm not sure.  I don't have that problem.

**Comment by sheriffderek** [comment]
Depending on your case, you could adjust the number you'll feed into the odometer before hand. [jsBin example](https://jsbin.com/cewusabeve/edit?js,console)
